v1alpha1/meshmodel.json

```diff
@@ -2027,6 +2147,114 @@
             }
           ]
         },
+        {
+          "name": "TrafficTarget",
+          "longName": "TrafficTarget",
+          "fullName": "easemesh.v1alpha1.TrafficTarget",
+          "description": "TrafficTarget is the specification of a TrafficTarget",
+          "hasExtensions": false,
+          "hasFields": true,
+          "hasOneofs": false,
+          "extensions": [],
+          "fields": [
+            {
+              "name": "name",
+              "description": "Name is the name for referencing a TrafficTarget",
+              "label": "",
+              "type": "string",
+              "longType": "string",
+              "fullType": "string",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "destination",
+              "description": "Destination is the service to allow ingress traffic",
+              "label": "",
+              "type": "IdentityBindingSubject",
+              "longType": "IdentityBindingSubject",
+              "fullType": "easemesh.v1alpha1.IdentityBindingSubject",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "sources",
+              "description": "Sources are the services to allow egress traffic",
+              "label": "repeated",
+              "type": "IdentityBindingSubject",
+              "longType": "IdentityBindingSubject",
+              "fullType": "easemesh.v1alpha1.IdentityBindingSubject",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "rules",
+              "description": "Rules are the traffic rules to allow (HTTPRoutes)",
+              "label": "repeated",
+              "type": "TrafficTargetRule",
+              "longType": "TrafficTargetRule",
+              "fullType": "easemesh.v1alpha1.TrafficTargetRule",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            }
+          ]
+        },
+        {
+          "name": "TrafficTargetRule",
+          "longName": "TrafficTargetRule",
+          "fullName": "easemesh.v1alpha1.TrafficTargetRule",
+          "description": "TrafficTargetRule is the TrafficSpec to allow for a TrafficTarget",
```

Comment:
What's is `TrafficSpec` I think it's a  dedicated word in the EaseMesh system, you should explain it in advance.